### ISSUE TITLE
[Nuxt 3.7.2] Error: Cannot read properties of undefined (reading 'use')

### ISSUE THREAD

**Issue by nathanchase** [prazdevs/pinia-plugin-persistedstate#239]
### Describe the bug

When running with Nuxt 3.7.2, an error occurs:
```
[nuxt] [request error] [unhandled] [500] Cannot read properties of undefined (reading 'use')
  at eval (./node_modules/@pinia-plugin-persistedstate/nuxt/dist/runtime/plugin.mjs:10:9)  
  at eval (./virtual:nuxt:/home/projects/github-xntdt3/.nuxt/plugins/server.mjs:36:89)  
  at Module.executeAsync (./node_modules/unctx/dist/index.mjs:130:19)  
  at setup (./virtual:nuxt:/home/projects/github-xntdt3/.nuxt/plugins/server.mjs:36:65)  
  at eval (./node_modules/nuxt/dist/app/nuxt.js:114:60)  
  at fn (./node_modules/nuxt/dist/app/nuxt.js:162:44)  
  at Object.callAsync (./node_modules/unctx/dist/index.mjs:87:55)  
  at eval (./node_modules/nuxt/dist/app/nuxt.js:164:56)  
  at Object.runWithContext (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:3855:18)  
  at callWithNuxt (./node_modules/nuxt/dist/app/nuxt.js:164:24)
  ```
Downgrading to Nuxt 3.7.1, it works: https://stackblitz.com/edit/github-xntdt3-fzhfhx?file=package.json

### Reproduction

https://stackblitz.com/edit/github-xntdt3?file=package.json

### System Info

```Shell
System:
    OS: Linux 5.15 Ubuntu 22.04.1 LTS 22.04.1 LTS (Jammy Jellyfish)
    CPU: (24) x64 AMD Ryzen 9 3900X 12-Core Processor
    Memory: 20.00 GB / 23.44 GB
    Container: Yes
    Shell: 5.8.1 - /usr/bin/zsh
  Binaries:
    Node: 18.15.0 - ~/.nvm/versions/node/v18.15.0/bin/node
  Browsers:
    Chrome: 111.0.5563.64
```
Crossposted: https://github.com/nuxt/nuxt/issues/23166

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/prazdevs/pinia-plugin-persistedstate/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/prazdevs/pinia-plugin-persistedstate/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by yeonjulee1005** [comment]
i'm also occur this issue!
please help me 😢

**Comment by martinszeltins** [comment]
Same issue for me, when I upgraded to `3.7.2` I got this error too 😞

**Comment by arunanshub** [comment]
Seems to work with `nuxt@3.6.5` and broken in `nuxt>=3.7`

**Comment by cdeclerck** [comment]
Same problem with `3.6.5` and upgrade to `3.7.1` didn't work for me

**Comment by nathanchase** [comment]
Upstream hotfix incoming! https://github.com/nuxt/nuxt/issues/23166#issuecomment-1717220105

**Comment by dosstx** [comment]
Thanks for writing the original issue, @nathanchase . Saved me some time and now I wait for that blessed hot fix....

**Comment by nathanchase** [comment]
This issue is resolved with [Nuxt 3.7.3](https://github.com/nuxt/nuxt/releases/tag/v3.7.3).

**Comment by jaderinoo** [comment]
Yup, looks like the issue is resolved.
